src/util/address.rs

```diff
@@ -98,6 +100,7 @@ impl fmt::Display for Error {
             Error::InvalidSegwitV0ProgramLength(l) => write!(f, "a v0 witness program must be either of length 20 or 32 bytes: length={}", l),
             Error::UncompressedPubkey => write!(f, "an uncompressed pubkey was used where it is not allowed"),
             Error::ExcessiveScriptSize => write!(f, "Script size exceed 520 bytes"),
+            Error::UnrecognizedScript => write!(f, "Script is not a p2pk, p2sh or witness program")
```

Comment:
Same as above. For the record, error messages should start with a lower case letter because they are often combined with other error messages e.g., "erorr one: error two: error three".

Reply:
Adding a commit to update the error message for Error::ExcessiveScriptSize.